ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -73,11 +71,10 @@
                 <span>Answer: @Model.AnswerToken</span>
             </div>
         }
-        else if (!@Model.Event.IsAnswerSubmissionActive)
+        else if (@Model.Event.EventBegin > DateTime.UtcNow)
```

Comment:
Nit: I think this is more readable the other way - DateTime < EventBegin
That reads closer to how you'd describe it in English:
* Yes: If it's before the start of the event / if now is before the start of the event
* Awkward: If the event starts after now / if the event start is after now